### DISCUSSION TITLE
Flag Converter in Hybrid Command

### DISCUSSION THREAD

**Discussion by insanephin** [Rapptz/discord.py#8259] [Q&A]
Flag Converter name(or rename) does not work in message commands
After defining the argument in English, it was changed to Korean through name. (user -> 유저)
but it doesn't work!

to express my thoughts
👍 : `!purge 3 유저: <@354184274347294720>`
👎 : `!purge 3 user: <@354184274347294720>`

did i do something wrong?

**Comment by Rapptz** [comment]
The `app_commands.rename` decorator clearly only works with application commands if this is what you're asking.

**Reply by insanephin** [reply]
Then, is there any other way to do it with (ctx, amount, 유저) instead of (ctx, amount, user)?

**Reply by insanephin** [reply]
Or should I use aliases?

**Reply by Rapptz** [reply]
You can pass aliases using this function https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#discord.ext.commands.flag

```py
class X(commands.FlagConverter):
    a: int = commands.flag(aliases=['b', 'c'])
```

**Reply by insanephin** [reply]
It's a personal opinion, but I think it would be nice to be able to replace aliases used by "name" or "rename"

Thank you for answer!
